pkg/cmd/pipeline/list_test.go

```diff
@@ -156,6 +156,114 @@ func TestPipelineList_only_pipelines_v1beta1(t *testing.T) {
 	golden.Assert(t, output, fmt.Sprintf("%s.golden", t.Name()))
 }
 
+func TestPipelineList_only_pipelines_no_headers_v1beta1(t *testing.T) {
+	pipelines := []pipelineDetails{
+		{"tomatoes", 1 * time.Minute},
+		{"mangoes", 20 * time.Second},
+		{"bananas", 512 * time.Hour}, // 3 weeks
+	}
+
+	nsList := []*corev1.Namespace{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "namespace",
+			},
+		},
+	}
+	version := "v1beta1"
+	clock := clockwork.NewFakeClock()
+	cs, pdata := seedPipelines(t, clock, pipelines, "namespace", nsList)
+	cs.Pipeline.Resources = cb.APIResourceList(version, "pipeline")
+	dc, err := testDynamic.Client(
+		cb.UnstructuredP(pdata[0], version),
+		cb.UnstructuredP(pdata[1], version),
+		cb.UnstructuredP(pdata[2], version),
+	)
+	if err != nil {
+		t.Errorf("unable to create dynamic clinet: %v", err)
+	}
+	p := &test.Params{Tekton: cs.Pipeline, Clock: clock, Kube: cs.Kube, Dynamic: dc}
+
+	pipeline := Command(p)
+	output, err := test.ExecuteCommand(pipeline, "list", "-n", "namespace", "--no-headers")
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+	golden.Assert(t, output, fmt.Sprintf("%s.golden", t.Name()))
+}
+
+func TestPipelineList_only_pipelines_all_namespaces_v1beta1(t *testing.T) {
+	pipelines := []pipelineDetails{
+		{"tomatoes", 1 * time.Minute},
+		{"mangoes", 20 * time.Second},
+		{"bananas", 512 * time.Hour}, // 3 weeks
+	}
+
+	nsList := []*corev1.Namespace{
```

Comment:
For this test, you should have multiple namespaces since this tests `--all-namespaces`.

Reply:
Here we cannot add more namespaces as the test fails with `Unexpected error: more than one object matched gvr /v1, Resource=namespaces, ns: "" name: ""`